### PR TITLE
Allow null value to getClaim method

### PR DESCRIPTION
**Pull request by joanfont**
See https://github.com/lcobucci/jwt/issues/249

### CODE REVIEW

File: test/unit/TokenTest.php [hunk 1]
```diff
@@ -221,6 +221,21 @@ public function getClaimShouldRaiseExceptionWhenClaimIsNotConfigured()
...
+     * @uses Lcobucci\JWT\Token::__construct
+     * @uses Lcobucci\JWT\Token::hasClaim
+     * @uses Lcobucci\JWT\Claim\Basic
+     *
+     * @covers Lcobucci\JWT\Token::getClaim
+     */
+    public function getClaimShouldReturnNullValueWhenDefaultParameterIsPassed()
+    {
+        $token = new Token();
+        $this->assertNull($token->getClaim('testing', null));
```
Comment (lcobucci) [inline test/unit/TokenTest.php:236]: Just a small nitpick, this method is static. Would you please update the call to `self::assertNull()`?

Comment (joanfont) [reply test/unit/TokenTest.php]: Sure, done :+1:

Comment (lcobucci) [reply test/unit/TokenTest.php]: Awesome, thanks!

**Comment by lcobucci** [comment]
@joanfont thank you for your contribution!

**Pull request merged**